requirements/static/ci/common.in

```diff
@@ -7,15 +7,17 @@ boto3>=1.16.0,<1.17.0; python_version < '3.6'
 boto3>=1.17.67; python_version >= '3.6'
 boto>=2.46.0
 cassandra-driver>=2.0
-certifi
+certifi>=2022.12.7; python_version >= "3.6"
+certifi<=2021.10.8; python_version <= "3.5"
```

Comment:
We don't run 3.5 anywhere these days...

Reply:
We still support 3.5 in 3005 though. I know this is just for the tests here, but I figured might as well.

Reply:
Oh, wait, this is the freeze branch.

Reply:
We also have certifi in some PKG requirents.

Reply:
https://github.com/saltstack/salt/commit/7df5feb62be4d18fc5c053d656e35b354430cda8

Reply:
Yeah, my pr is adding it to some of the pkg ones as well. Is there concern about this?

Reply:
No, no. Didn't saw them and didn't want them to get missed.